framework_tvb/tvb/interfaces/rest/client/examples/launch_operation.py

```diff
@@ -0,0 +1,167 @@
+# -*- coding: utf-8 -*-
+#
+#
+# TheVirtualBrain-Framework Package. This package holds all Data Management, and
+# Web-UI helpful to run brain-simulations. To use it, you also need do download
+# TheVirtualBrain-Scientific Package (for simulators). See content of the
+# documentation-folder for more details. See also http://www.thevirtualbrain.org
+#
+# (c) 2012-2020, Baycrest Centre for Geriatric Care ("Baycrest") and others
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software Foundation,
+# either version 3 of the License, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+# You should have received a copy of the GNU General Public License along with this
+# program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+#   CITATION:
+# When using The Virtual Brain for scientific publications, please cite it as follows:
+#
+#   Paula Sanz Leon, Stuart A. Knock, M. Marmaduke Woodman, Lia Domide,
+#   Jochen Mersmann, Anthony R. McIntosh, Viktor Jirsa (2013)
+#       The Virtual Brain: a simulator of primate brain network dynamics.
+#   Frontiers in Neuroinformatics (7:10. doi: 10.3389/fninf.2013.00010)
+#
+#
+
+from uuid import UUID
+import time
+import tvb_data
+import os
+from tvb.adapters.uploaders.csv_connectivity_importer import CSVConnectivityImporterModel
+from tvb.adapters.uploaders.region_mapping_importer import RegionMappingImporterModel
+from tvb.adapters.uploaders.zip_connectivity_importer import ZIPConnectivityImporterModel
+from tvb.adapters.uploaders.zip_surface_importer import ZIPSurfaceImporterModel
+from tvb.basic.logger.builder import get_logger
+from tvb.basic.profile import TvbProfile
+from tvb.config.init.datatypes_registry import populate_datatypes_registry
+from tvb.core.entities.model.model_operation import STATUS_ERROR, STATUS_CANCELED, STATUS_FINISHED
+from tvb.core.entities.storage import dao
+from tvb.core.neocom import h5
+from tvb.core.neotraits._h5core import ViewModelH5
+from tvb.interfaces.rest.client.tvb_client import TVBClient
+
+if __name__ == '__main__':
+    # TODO: remove this line after pypi package update
+    populate_datatypes_registry()
```

Comment:
This line (and the import) should be removed, as it is already present in the TVBClient